applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -41,6 +41,12 @@ The Managed services option is also available if the RHOAS Operator is installed
 ** *Import YAML*: Upload a YAML file to create and define resources for building and deploying applications.
 ** *Upload JAR file*: Upload a JAR file to build and deploy Java applications.
 
+* *Share my Project*: Use this option to add or remove users to a project and provide accessibility options to them.
+
+* *Helm Chart repositories*: Use this option to add Helm Chart repositories in a namespace.
+
+* *Re-ordering of resources*: Allows the user to re-order pinned resources added to the navigation pane. The drag-and-drop icon is displayed on the left side of the pinned resource when you hover over it in the navigation pane. The dragged resource can be dropped only in the section where it resides.
```

Comment:
For all your other descriptions, you use the second person. So use second person here as well. 

`Use these resources to re-order pinned resources added to your navigation pane.` 

The rest of your content is good :) The first sentence threw me off since it says `allows users` then you go to second person for the remaining sentences.